zaza/utilities/deployment_env.py

```diff
@@ -106,12 +108,18 @@ def get_overlay_ppas():
         return config.overlay_ppas
     except KeyError:
         pass
+    try:
+        return config[model_alias].overlay_ppas
```

Comment:
shouldn't we try the model_alias overlay_ppas first and on KeyError fall back to the "top level" overlay ppas? or may not fallback at all?

Reply:
Yes that's a good idea. I'll make an update.